### ISSUE TITLE
Food Scraps Accounts - need mailing address

### ISSUE THREAD

**Issue by BPS-PCW** [BPSTechServices/swr-public#11]
Hey Derek, 

Can you add a mailing address to the food scraps businesses? We've run into some accounts that have different addresses for physical and need an additional space for mailing address. Thank you, Pete

**Comment by derekwmiller** [comment]
@BPS-PCW Yes. I can append mailing address to the table. What should I populate it with initially?

**Comment by BPS-PCW** [comment]
Nothing, we have 7 accounts that have PO boxes, so we'll need them occasionally to make sure they get our mailings and not lose the physical addresses for site visits.


Pete Chism-Winfield (he/him)
Policy and Compliance Coordinator



Garbage and Recycling Team

City of Portland Bureau of Planning and Sustainability
Cell: 503-250-2081 | ***@***.***
1810 SW 5th Ave., Suite 710 Portland, OR 97201



Office hours: Mon, 930 AM - 530 PM, Tues & Thurs, 7 AM - 5PM, Wed, 630 AM - 230 PM, Friday, 12 PM – 430 PM
Website: Garbage, Recycling, and Compost<https://www.portland.gov/bps/garbage-recycling> | @portlandbps<https://www.twitter.com/portlandbps> | subscribe to e-news<https://www.portlandoregon.gov/bps/article/404078>



The City of Portland is committed to providing meaningful access. For accommodations, modifications, translation, interpretation or other services<https://www.portlandoregon.gov/bps/71701>, please contact at 503-823-7700 or use City TTY 503-823-6868.

________________________________
